tests/test_relation_u.py

```diff
@@ -5,83 +5,86 @@
 from .schema_simple import *
 
 
-class TestU:
-    """
-    Test tables: insert, delete
-    """
-
-    @classmethod
-    def setup_class(cls):
-        cls.user = User()
-        cls.language = Language()
-        cls.subject = Subject()
-        cls.experiment = Experiment()
-        cls.trial = Trial()
-        cls.ephys = Ephys()
-        cls.channel = Ephys.Channel()
-        cls.img = Image()
-        cls.trash = UberTrash()
```

Comment:
Instead of `setup_class`, these tests use fixtures like `subject` and `experiment` which are now defined in `conftest.py`.